.github/workflows/IntegrationTest.yml

```diff
@@ -2,18 +2,18 @@ name: Spark Connector Integration Test
 
 on:
   push:
-    branches: [ master ]
+    branches: [ spark_2.3 ]
   pull_request:
-    branches: [ master ]
+    branches: [ spark_2.3 ]
 
 jobs:
   test:
     name: IT
     runs-on: ubuntu-latest
     strategy:
      matrix:
-       scala_version: [ '2.12.11' ]
-       spark_version: [ '3.0.0' ]
+       scala_version: [ '2.11.12' ]
+       spark_version: [ '2.3.0' ]
```

Comment:
2.3.0 over here, 2.3.4 above. Is this intentional?

Reply:
Yes. It is intentional. The IT test is run for spark 2.3.0/2.4.0/3.0.0. We didn't actually run for all released versions.
When I work on Cluster Test, I used some new versions: 2.3.4/2.4.6/3.0.0.

Since the SPARK should keep the interface consistent. I believe it is not a problem. And we didn't get any customer to report this kind of compatibility issue.